### ISSUE TITLE
easier setup with docker compose?

### ISSUE THREAD

**Issue by sebastiaan101** [EHDEN/ARACHNE#42]
Suggestion perhaps to use docker compose.
easier, with port numbers etc in parameters.

**Comment by konstjar** [comment]
Any contribution is welcome for docker compose scripts.

**Comment by PRijnbeek** [comment]
Agree. let's keep it open till we have it. Multiple people already tried this.
